src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/speaker/SpeakerRulesRouter.java

```diff
@@ -1,98 +0,0 @@
- *   See the License for the specific language governing permissions and
- *   limitations under the License.
- */
-
-package org.openkilda.wfm.topology.network.storm.bolt.speaker;
-
-import org.openkilda.messaging.Message;
-import org.openkilda.messaging.info.InfoData;
-import org.openkilda.messaging.info.InfoMessage;
-import org.openkilda.messaging.info.discovery.InstallIslDefaultRulesResult;
-import org.openkilda.messaging.info.discovery.RemoveIslDefaultRulesResult;
-import org.openkilda.wfm.AbstractBolt;
-import org.openkilda.wfm.error.PipelineException;
-import org.openkilda.wfm.share.model.IslReference;
-import org.openkilda.wfm.topology.network.storm.ComponentId;
-import org.openkilda.wfm.topology.network.storm.bolt.isl.command.IslCommand;
-import org.openkilda.wfm.topology.network.storm.bolt.speaker.command.SpeakerRulesIslInstalledCommand;
-import org.openkilda.wfm.topology.network.storm.bolt.speaker.command.SpeakerRulesIslRemovedCommand;
-import org.openkilda.wfm.topology.network.storm.bolt.speaker.command.SpeakerRulesWorkerCommand;
-import org.openkilda.wfm.topology.utils.MessageKafkaTranslator;
-
-import lombok.extern.slf4j.Slf4j;
-import org.apache.storm.topology.OutputFieldsDeclarer;
-import org.apache.storm.tuple.Fields;
-import org.apache.storm.tuple.Tuple;
-import org.apache.storm.tuple.Values;
-
-@Slf4j
-public class SpeakerRulesRouter extends AbstractBolt {
-    public static final String BOLT_ID = ComponentId.SPEAKER_RULES_ROUTER.toString();
-
-    public static final String FIELD_ID_KEY = MessageKafkaTranslator.FIELD_ID_KEY;
-    public static final String FIELD_ID_INPUT = MessageKafkaTranslator.FIELD_ID_PAYLOAD;
-
-    public static final String STREAM_WORKER_ID = "worker";
-    public static final Fields STREAM_WORKER_FIELDS = new Fields(FIELD_ID_KEY, FIELD_ID_INPUT, FIELD_ID_CONTEXT);
-
-    @Override
-    protected void handleInput(Tuple input) throws Exception {
-        String source = input.getSourceComponent();
-        if (ComponentId.INPUT_SPEAKER_RULES.toString().equals(source)) {
-            Message message = pullValue(input, FIELD_ID_INPUT, Message.class);
-            speakerMessage(input, message);
-        } else {
-            unhandledInput(input);
-        }
-    }
-
-    private void speakerMessage(Tuple input, Message message) throws PipelineException {
-        proxySpeaker(input, message);
-    }
-
-    private void proxySpeaker(Tuple input, Message message) throws PipelineException {
-        if (message instanceof InfoMessage) {
-            proxySpeaker(input, ((InfoMessage) message).getData());
-        } else {
-            log.debug("Do not proxy speaker message - unexpected message type \"{}\"", message.getClass());
-        }
-    }
-
-    private void proxySpeaker(Tuple input, InfoData payload) throws PipelineException {
-        if (payload instanceof InstallIslDefaultRulesResult) {
-            emit(STREAM_WORKER_ID, input, makeWorkerTuple(new SpeakerRulesIslInstalledCommand(
```

Comment:
Who will encode data into network stream commands? Why this bolt was removed?

Reply:
Returned it back.